wp-admin/js/widgets/media-widgets.js

```diff
@@ -57,54 +60,79 @@ wp.mediaWidgets = ( function( $ ) {
 				}
 			}
 
-			control.attachmentFetched = $.Deferred();
-			control.selectedAttachment = new wp.media.model.Attachment();
-			if ( control.model.get( 'attachment_id' ) ) {
-				control.selectedAttachment.set( {
-					id: control.model.get( 'attachment_id' )
-				} );
-				control.selectedAttachment.fetch().done( function() {
-					control.attachmentFetched.resolve();
-				} );
-			}
+			// Re-render the preview when the attachment changes.
+			control.selectedAttachment = new wp.media.model.Attachment( { id: 0 } );
+			control.listenTo( control.selectedAttachment, 'change', function() {
```

Comment:
Good point! Yes, if we add `renderPreview` up above to the `bindAll` call. I'll do that.